pkg/reconciler/instances/ory/action_test.go

```diff
@@ -517,7 +491,87 @@ func TestOryDbSecret(t *testing.T) {
 		})
 	}
 }
+func TestOryDbSecret_IsUpdate(t *testing.T) {
```

Comment:
I have a problem with this testing function. Is it what you really want to test? The implementation of the function is as simple as this:
```go
func isUpdate(diff map[string]string) bool {
	return len(diff) != 0
}
```
I would imagine test scenarios where the tested function retrieves some crafted input of type `map[string]string` and nothing else. Whereas here you do many things, including retrieval, creation and update of a secret. If it was supposed to be the integration test for the action (which one in fact?) then you should reconsider the test name and not imply that what is really tested is `isUpdate()` function because now it is not. If you unit test it in some other place you could in fact use it as an assertion and expect that it says that the updated is needed or not. For now I feel a mixed responsibility of the testing function. Could you somehow adjust it?

Reply:
I think I wanted to test the update logic, because there were no such tests in place before. I will rename this function to not being considered as a test for the isUpdate method.